src/libbluechi/common/string-util.c

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
Please add a unit test with a few cases for the string builder to
https://github.com/eclipse-bluechi/bluechi/tree/main/src/libbluechi/test/common/string-util